### ISSUE TITLE
oxygen tanks or somethin

### ISSUE THREAD

**Issue by cyber63** [studio-minus/ppg-bugs-and-requests#1085]
What if you get the blood tank without any fluid in it and hook it up to a humans head or torso select push mode and then if he goes underwater with it he will not drown, it saves you from injecting them with a syringe then stabbing them underwater and then getting a heal syringe to fix them. Or just add oxygen tanks to the game.

**Comment by Drogo-man** [comment]
Water breathing syring

**Comment by DindinYT37** [comment]
> Water breathing syring

Yeah but you can't undo it and the human cannot run out of the oxygen like that.

**Comment by Drogo-man** [comment]
> > Water breathing syring
> 
> Yeah but you can't undo it and the human cannot run out of the oxygen like that.

Yeah, that's really the only problem. It's not perfect, but it's the only current solution.

**Comment by DindinYT37** [comment]
> > > Water breathing syring
> > 
> > 
> > Yeah but you can't undo it and the human cannot run out of the oxygen like that.
> 
> Yeah, that's really the only problem. It's not perfect, but it's the only current solution.

That's why he requested this 👌

**Comment by Drogo-man** [comment]
> > > > Water breathing syring
> > > 
> > > 
> > > Yeah but you can't undo it and the human cannot run out of the oxygen like that.
> > 
> > Yeah, that's really the only problem. It's not perfect, but it's the only current solution.
> 
> That's why he requested this 👌

But I think it would be better as a separate item, not just an empty blood tank.

**Comment by DindinYT37** [comment]
> > > > > Water breathing syring
> > > > 
> > > > 
> > > > Yeah but you can't undo it and the human cannot run out of the oxygen like that.
> > > 
> > > 
> > > Yeah, that's really the only problem. It's not perfect, but it's the only current solution.
> > 
> > 
> > That's why he requested this 👌
> 
> But I think it would be better as a separate item, not just an empty blood tank.

It seems you're looking for #787 then. Also @cyber63 mentioned "Or just add oxygen tanks to the game." at the end of the issue - probably meaning he wasn't sure of how to give the oxygen to the human, he just wanted a way to do so.
